pkg/operator/ceph/cluster/mon/spec.go

```diff
@@ -215,6 +216,26 @@ func (c *Cluster) makeChownDataDirInitContainer(monConfig *monConfig) v1.Contain
 	return container
 }
 
+func (c *Cluster) makeChownLogDirInitContainer(monConfig *monConfig) v1.Container {
+	container := v1.Container{
+		Name: "chown-container-log-dir",
+		Command: []string{
+			"chown",
```

Comment:
I believe `chown` will take multiple targets. Can we add `config.VarLogCephDir` to the existing chown inside `makeChownDataDirInitContainer()` (and rename that method)?

Reply:
It will take multiple targets, though one instance is recursive and the other is not. I suspect non-recursive was used for the data dir in case filestore was being used there could be a lot of file updates. Does that matter anymore?

Reply:
If this is specific to mons, filestore won't matter, right? The daemons will also be running as the `ceph` user, so I believe all files created after the first launch would not need to be chowned again anyway.

Reply:
ah :) yes filestore does indeed not matter.